Add database

PakSemen opened the pull request.

PakSemen committed:
Add files via upload

PakSemen committed:
Update README.md

PakSemen committed:
Update README.md

PakSemen committed:
Update README.md

PakSemen committed:
Update README.md

PakSemen committed:
Update README.md

PakSemen committed:
Update README.md

PakSemen committed:
Delete Untitled.png

PakSemen committed:
Add files via upload

PakSemen committed:
Update README.md

PakSemen committed:
Update README.md

AlekseiZubkov reviewed (commented):
MPA  лучше вынести в отдельную таблицу

PakSemen committed:
Update README.md

PakSemen committed:
Delete Sheme.png

PakSemen committed:
Add files via upload

PakSemen committed:
Delete Shema.png

PakSemen committed:
Add files via upload

PakSemen committed:
Update README.md

сменил названия некоторых таблиц, чтобы они не совпадали со служебными словами Sql

PakSemen committed:
Delete Shema.png

PakSemen committed:
Add files via upload

PakSemen committed:
Delete Shema.png

PakSemen committed:
Add files via upload

PakSemen committed:
Update README.md

PakSemen committed:
Update README.md

PakSemen committed:
Delete Shem.png

PakSemen committed:
Add files via upload

PakSemen committed:
feature:добавлена работа с БД, созданы ORM механизмы, для фильмов добавлены поля с возрастным рейтингом и жанрами

PakSemen committed:
Merge branch 'main' into add-database

PakSemen committed:
Update UserService.java

PakSemen committed:
Update FilmDao.java

PakSemen committed:
Update UserDao.java

PakSemen committed:
Update InMemoryUserStorage.java

PakSemen committed:
Update UserStorage.java

PakSemen committed:
Update InMemoryUserStorage.java

PakSemen committed:
refactor: выделил работу с друзьями и лайками в отдельный класс, убрал ненужные метод и сделал запросы к БД одиночными, где это нужно

PakSemen committed:
Merge remote-tracking branch 'origin/add-database' into add-database

PakSemen committed:
refactor: выделил работу с друзьями и лайками в отдельный класс, убрал ненужные метод и сделал запросы к БД одиночными, где это нужно

PakSemen committed:
refactor: выделил работу с друзьями и лайками в отдельный класс, убрал ненужные метод и сделал запросы к БД одиночными, где это нужно

PakSemen committed:
refactor: убрал все лишние методы из MpaDaoImpl и GenreDaoImpl, переместил сохранение фильмов полностью в FilmDaoImpl
